be/src/vec/exec/scan/scanner_context.h

```diff
@@ -225,6 +225,8 @@ class ScannerContext : public std::enable_shared_from_this<ScannerContext>,
     const int _num_parallel_instances;
     std::shared_ptr<RuntimeProfile> _scanner_profile;
     RuntimeProfile::Counter* _scanner_sched_counter = nullptr;
+    // This counter refers to scan operator's local state
+    RuntimeProfile::Counter* _memory_used_counter = nullptr;
```

Comment:
`_scan_operator_memory_used_counter` is better

Reply:
renamed to _scanner_memory_used_counter like _scanner_peak_memory_usage